cmd/cli/serve/serve.go

```diff
@@ -571,3 +562,51 @@ func pickP2pAddress(addresses []multiaddr.Multiaddr) multiaddr.Multiaddr {
 
 	return addresses[0]
 }
+func GetTLSCertificate(nodeConfig *node.NodeConfig, cert string, key string) (string, string, error) {
+
+	if nodeConfig.RequesterAutoCert != "" {
+		return "", "", nil
+	}
```

Comment:
Worth a comment to explain this,.
```suggestion
        // If there are configuration values for autocert we should return an let autocert 
        // do what it does later on in the setup.
	if nodeConfig.RequesterAutoCert != "" {
		return "", "", nil
	}
```